src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -114,7 +114,7 @@
 (defn- do-with-jdbc-transaction [database-id f]
   (if *connection*
     (f *connection*)
-    (let [jdbc-spec (sql-jdbc.conn/db->pooled-connection-spec database-id)]
+    (let [jdbc-spec (sql-jdbc.conn/db+connection-type->pooled-connection-spec database-id ::sql-jdbc.conn/rw)]
```

Comment:
I don't think there's a really great reason to use namespaced keywords here, no one is going to get confused if you use plain keywords. I'd rather use something like `:read` or `:readwrite` than `::sql-jdbc.conn/r` and `::sql-jdbc.conn/rw`

Reply:
i think there's some good arguments to use a namespaced keyword, but I think autoresolved keywords aren't a good fit. `:conn/r` and `:conn/rw` are nice and succint. I am a big fan of finding all usages of keywords so finding `:conn/` as a prefix or `:conn/r` help me find how this option is used everywhere. `:read` already exists as a permission and so it is very hard to find how it is used. (other options are `:pool/r` or `:spec/r`. I think there's good reason for a namespace but `:metabase.driver.sql-jdbc.actions` isn't a good prefix.)